libs/angular-components/community/components/tags/status-badge/status-badge.component.scss

```diff
@@ -0,0 +1,133 @@
+// Identical to libs/react-components/src/tedi/components/tags/status-badge/status-badge.module.scss,
+//  redo to use shared file when possible
+@use "@tehik-ee/tedi-core/mixins";
+
+$badge-colors: (
+  "neutral",
+  "brand",
+  "accent",
+  "success",
+  "danger",
+  "warning",
+  "transparent"
+);
+$badge-variants: (
+  "filled",
+  "filled-bordered",
+  "bordered"
+);
+$badge-status-colors: (
+  "inactive",
+  "success",
+  "danger",
+  "warning"
+);
+
+:root {
+  --status-badge-icon-primary: var(--blue-700);
+  --status-badge-icon-secondary: var(--orange-700);
+}
+
+.tedi-status-badge {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+
+  @include mixins.responsive-styles(font-size,
+    body-small-regular-size,
+    $exclude: tablet);
+  @include mixins.responsive-styles(font-weight,
+    body-small-regular-weight,
+    $exclude: tablet);
+  @include mixins.responsive-styles(line-height,
+    body-small-regular-line-height,
+    $exclude: tablet);
+  @include mixins.responsive-styles(padding,
+    tag-default-padding-y tag-default-padding-x);
+  @include mixins.responsive-styles(border-radius, tag-default-radius);
+
+  &__inner-text {
+    min-width: fit-content;
+
+    @include mixins.responsive-styles(padding-left, tag-default-padding-x);
+    @include mixins.responsive-styles(padding-right, tag-default-padding-x);
+  }
+
+  &--large {
+    @include mixins.responsive-styles(min-width, tag-status-large-min-width);
+    @include mixins.responsive-styles(padding,
+      tag-status-large-padding-y tag-status-large-padding-x);
+  }
+
+  @each $variant in $badge-variants {
+    &.tedi-status-badge--variant-#{$variant} {
+      @each $color in $badge-colors {
+        &.tedi-status-badge--color-#{$color} {
+          color: var(--status-badge-text-#{$color});
```

Comment:
maybe one thing i would change compared to react is to assign the values to private css variable to keep the same pattern with other angular components